### PR TITLE
Sortable: describe update method

### PR DESCRIPTION
**Pull request by pavel-2020**

### CODE REVIEW

File: api-reference/10 UI Widgets/dxSortable/3 Methods/update().md [hunk 1]
```diff
@@ -3,7 +3,18 @@ id: dxSortable.update()
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Updates **Sortable**'s dimensions. Call this method after items are added or their dimensions are changed.
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxSortable/3 Methods/update().md]:
```suggestion
Updates **Sortable**'s dimensions. Call this method after items are added or their dimensions are changed during dragging.
```

File: api-reference/10 UI Widgets/dxSortable/3 Methods/update().md [hunk 2]
```diff
@@ -3,7 +3,18 @@ id: dxSortable.update()
...
 ---
-<!-- Description goes here -->
\ No newline at end of file
+Depending on the framework, use this method in different places:
+
+* **jQuery** - immediately after a markup update;
+
+* **React** - in the <a href="https://reactjs.org/docs/hooks-reference.html#useeffect" target="_blank">useEffect</a> hook or in the <a href="https://reactjs.org/docs/react-component.html#componentdidupdate" target="_blank">componentDidUpdate</a> lifecycle method;
+
+* **Angular** - in the ngAfterContentChecked hook;
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxSortable/3 Methods/update().md]:
```suggestion
* **Angular** - in the `ngAfterContentChecked` hook;
```

File: api-reference/10 UI Widgets/dxSortable/3 Methods/update().md [hunk 3]
```diff
@@ -3,7 +3,18 @@ id: dxSortable.update()
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Updates **Sortable**'s dimensions. Call this method after items are added or their dimensions are changed during dragging.
```
Comment (albertov05) [inline api-reference/10 UI Widgets/dxSortable/3 Methods/update().md:6]: Is "Sortable" the method name?
Can you clarify how this method works with dimensions, for my understanding?

File: api-reference/10 UI Widgets/dxSortable/3 Methods/update().md [hunk 4]
```diff
@@ -3,7 +3,18 @@ id: dxSortable.update()
...
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+Depending on the framework, use this method in different places:
```
Comment (albertov05) [inline api-reference/10 UI Widgets/dxSortable/3 Methods/update().md]: I don't think "after a markup update" is a place. It sounds more like a point in time.
Can we make this statement more generic? For example:

`Depending on the framework, use this method as follows:`

File: api-reference/10 UI Widgets/dxSortable/3 Methods/update().md (hunk 3 above)
Comment (pavel-2020) [reply api-reference/10 UI Widgets/dxSortable/3 Methods/update().md:6]: Sortable is a widget. Here's the demo;
https://js.devexpress.com/Demos/WidgetsGallery/Demo/Sortable/Kanban/Vue/Light/
In that demo, the dimensions are the layout of one column and its inner elements.

File: api-reference/10 UI Widgets/dxSortable/3 Methods/update().md (hunk 4 above)
Comment (pavel-2020) [reply api-reference/10 UI Widgets/dxSortable/3 Methods/update().md]: Yes, it is better.

**Review by RomanTsukanov — APPROVED** [review]

File: api-reference/10 UI Widgets/dxSortable/3 Methods/update().md (hunk 3 above)
Comment (albertov05) [reply api-reference/10 UI Widgets/dxSortable/3 Methods/update().md:6]: Thank you. So, when I drag a column to a different position, the size (i.e., dimensions) seem to stay the same. Can you clarify how the dimension changes?

File: api-reference/10 UI Widgets/dxSortable/3 Methods/update().md (hunk 4 above)
Comment (albertov05) [reply api-reference/10 UI Widgets/dxSortable/3 Methods/update().md]: Thank you.

**Pull request merged**